iomgr/lib/dbengine/handlers/RequestHandler_Common.cpp

```diff
@@ -77,6 +77,11 @@ void RequestHandler::executeRequest(const requests::DBEngineRequest& request,
                         response, dynamic_cast<const requests::ShowDatabasesRequest&>(request));
                 break;
             }
+            case requests::DBEngineRequestType::kShowTables: {
```

Comment:
@niolap 
Remove this "case" and remove constant "kShowTables" - they never work and not needed. The whole idea of this constant is to identify each unique data structure. In this case you have declared constant, but you don't have unique data structure which uses it as own identifier. So there will work another identifier - "kSelect".